test/Mvc6Framework45.FunctionalTests/Startup.cs

```diff
@@ -124,8 +123,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             {
                 routes.MapRoute(
                     name: "default",
-                    template: "{controller}/{action}/{id?}",
-                    defaults: new { controller = "Home", action = "Index" });
+                    template: "{controller=Home}/{actionIndex}/{id?}");
```

Comment:
Tests are not running at all as of now. That is where we are stuck. The test applications compile fine, however the requests are not going out. We changed this recently, as we couldn't figure out the problem. Will fix this and see if this could be the reason behind the failure of test application. Thank you!